docs/core/diagnostics/debug-memory-leak.md

```diff
@@ -0,0 +1,243 @@
+---
+title: Debug a memory leak tutorial
+description: A tutorial walk-through, debugging a memory leak in .NET Core.
+author: sdmaclea
+ms.author: stmaclea
+ms.topic: tutorial
+ms.date: 11/24/2019
+---
+# Tutorial: Debugging a memory leak
+
+**This article applies to: ✓** .NET Core 3.0 SDK and later versions
+
+This tutorial demonstrates the tools to analyze a .NET Core memory leak.
+
+This tutorial uses a sample app, which is designed to intentionally leak memory. The sample is provided as an exercise. You can analyze an app that is unintentionally leaking memory too.
+
+In this tutorial, you will:
+
+> [!div class="checklist"]
+>
+> * Examine managed memory usage with [dotnet-counters](dotnet-counters.md)
```

Comment:
```suggestion
> * Examine managed memory usage with [dotnet-counters](dotnet-counters.md).
```

Reply:
@mairaw, Asterisks are used here for the list. And later on, the dash was used. I remember that linter complained about that. But the markdown lint check has passed here. The violated rule is [MD004](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md004---unordered-list-style)

Reply:
Fixed both.